syntaxes/gdb.tmLanguage.json

```diff
@@ -869,7 +869,7 @@
 			"patterns": [
 				{
 					"name": "meta.on_off_unlimited.gdb",
-					"match": "(u(?:n(?:l(?:i(?:m(?:i(?:t(?:e(?:d)?)?)?)?)?)?)?)|on|of+)",
+					"match": "(u(?:n(?:l(?:i(?:m(?:i(?:t(?:e(?:d)?)?)?)?)?)?)?)|on|off?)",
```

Comment:
best would be `off?\\b`, otherwise offffff will renders with off as blue and the excedent ffff in white.
We need to fix off anywhere its used, no only for this patterns.

Reply:
totally agree, let's close this PR and consider it as "issue" :-)